pkgs/by-name/ff/ffizer/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  libgit2,
+  openssl,
+  zlib,
+  stdenv,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ffizer";
+  version = "2.13.0";
+
+  buildFeatures = [ "cli" ];
+
+  src = fetchFromGitHub {
+    owner = "ffizer";
+    repo = "ffizer";
+    rev = version;
+    hash = "sha256-Kgd27X9t/AkNCkVWCHdNO6MJBLchxrPQkkmqEJVTrPo=";
+  };
+
+  cargoHash = "sha256-NTwhx0glxIrgTfx8K6tPXokL1ps5PsQ8rjEXThqHkzY=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs =
+    [
+      libgit2
+      openssl
+      zlib
+    ]
+    ++ lib.optionals stdenv.isDarwin [
+      darwin.apple_sdk.frameworks.Security
+    ];
+
+  env.OPENSSL_NO_VENDOR = true;
+
+  checkFlags = [
+    # requires internet access
+    "--skip=run_test_samples_tests_data_template_2"
+  ];
+
+  meta = {
+    description = "Ffizer is a files and folders initializer / generator. Create any kind (or part) of project from template";
```

```suggestion
    description = "Files and folders initializer / generator based on templates";
```
description is one sentence fragment.